apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -872,8 +861,8 @@ export const StepBasedFlow: FunctionComponent<AuthenticationFlowPropsInterface>
                 allowSocialLoginAddition={ true }
                 currentStep={ authenticatorAddStep }
                 open={ showAuthenticatorAddModal }
-                onModalSubmit={ (authenticators) => {
-                    authenticators.map((authenticator) => {
+                onModalSubmit={ (authenticators: any) => {
```

Comment:
Should avoid the use of any whenever possible since that'll defeat the purpose of Typescript.

```suggesion
onModalSubmit={ (authenticators: GenericAuthenticatorInterface) => {
```